mqr/utils.py

```diff
@@ -0,0 +1,6 @@
+from temba_client.v2 import TembaClient
+
+import vaccine.healthcheck_config as config
+
+
+rapidpro = TembaClient(config.RAPIDPRO_URL, config.RAPIDPRO_TOKEN)
```

Comment:
Doesn't look like we are actually using any of this, can remove this file.

Reply:
I am using it to get rapidpro contact

Reply:
I see, then we probably don't need the `def get_rapidpro():` function. 

Just not sure if `TembaClient` will work in async. But lets see

Reply:
You can then also move the:
`if config.RAPIDPRO_URL and config.RAPIDPRO_TOKEN:`
to here.

and replace it with `if rapidpro` where you are using it.